zenon/regression/examples/data/TLAPS.tla

```diff
@@ -1 +1 @@
-../TLAPS.tla
\ No newline at end of file
+../../../../library/TLAPS.tla
```

Comment:
I think this symlink should be removed rather than updated.

Reply:
Removed it.